WordPress/src/main/res/values/reader_styles.xml

```diff
@@ -211,7 +211,7 @@
 
     <!-- chips -->
     <style name="ReaderInterestFilterChip" parent="@style/Widget.MaterialComponents.Chip.Filter">
-        <item name="android:textColor">@color/on_surface_surface_selector</item>
+        <item name="android:textColor">@color/on_surface_on_secondary_selector</item>
```

Comment:
I'm wondering if it's ok to use "secondary_selector" for a button which uses "primaryColor". Wdyt?

Reply:
You're right, it doesn't seem ok. I checked we've `colorOnPrimarySurface` in addition to `colorOnPrimary`

`colorOnPrimary` => light mode = white, dark mode = black
`colorOnPrimarySurface` => light mode = white, dark mode = white

`colorOnPrimarySurface` serves our purpose so switching to it in 6b0efa3

Reply:
Awesome, thanks!